src/routes/about/+page.svelte

```diff
@@ -11,9 +11,13 @@
     import { biblesEndpoint, biblesWithRestrictionsEndpoint, parentResourcesEndpoint } from '$lib/api-endpoints';
     import { filterBoolean } from '$lib/utils/array';
     import PublicDomainIcon from '$lib/icons/PublicDomainIcon.svelte';
+    import { supportedLanguages } from '$lib/stores/language.store';
 
     let licenseInfosPromise: Promise<ApiLicenseInfo[]> | null = null;
 
+    let publicDomainMark =
```

Comment:
One nitpick from me. We store all Icons here: `src/lib/icons.` Any file in that directory is an easy example.

Reply:
I can't use the file, it's going to be an html string in the content of the translated string, so had to use the spelled out svg in the string

Reply:
When I use `<PublicDomainIcon />` it only outputs `PublicDomain` in the translated string, not the icon